Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1475,24 +1478,24 @@
         "com.apple.WebKit.showMemoryCache"
         "com.apple.WebKit.showPaintOrderTree"
         "com.apple.WebKit.showRenderTree"
-        "com.apple.accessibility.cache.app.ax"
-        "com.apple.accessibility.cache.ax"
-        "com.apple.accessibility.cache.enhance.text.legibility"
-        "com.apple.accessibility.cache.enhance.text.legibilitycom.apple.WebKit.WebContent"
-        "com.apple.accessibility.cache.guided.access.via.mdm"
-        "com.apple.accessibility.cache.invert.colors"
-        "com.apple.accessibility.cache.invert.colorscom.apple.WebKit.WebContent"
-        "com.apple.accessibility.cache.vot"
-        "com.apple.accessibility.cache.zoom"
+        "com.apple.analyticsd.running"
+        "com.apple.coreaudio.list_components"
+        "com.apple.distnote.locale_changed"
         "com.apple.language.changed"
         "com.apple.mediaaccessibility.audibleMediaSettingsChanged"
         "com.apple.mediaaccessibility.captionAppearanceSettingsChanged"
-        "com.apple.mobile.keybagd.lock_status"
-        "com.apple.mobile.keybagd.user_changed"
-        "com.apple.mobile.usermanagerd.foregrounduser_changed"
         "com.apple.powerlog.state_changed"
+        "com.apple.sessionagent.screenLockUIIsHidden"
+        "com.apple.sessionagent.screenLockUIIsShowing"
+        "com.apple.sessionagent.screenLockUIIsShown"
+        "com.apple.sessionagent.shieldWindowIsShowing"
+        "com.apple.sessionagent.shieldWindowLowered"
+        "com.apple.sessionagent.shieldWindowRaised"
```

Comment:
Are these in use on iOS?

Reply:
Not sure, my guess is possibly not because it is coming from `_NSAccessibilitySafeToVendElement` which is in AppKit.